.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish NPM package
+
+on:
+  workflow_run:
+    workflows: ["Checks"]
+    branches: [main]
+    types:
+      - completed
+
+jobs:
+  deploy:
+    name: Publish to NPM
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+
+      - name: Cache dependencies
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./node_modules
+          key: modules-${{ hashFiles('yarn.lock') }}
+
+      - name: Install yarn dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: rm -rf node_modules && yarn install --frozen-lockfile
```

Comment:
Why do we need dependencies installed just to publish?

(Legit question, not a veiled "you should remove this")

Reply:
I looked at how other people did this, and they always seem to install the dependencies first - presumably to have a final check that everything is still available before publishing. But it also works without it.

Given that we already check the dependencies in the tests (and because this uses a cached version anyways), I meant to remove it again for cleanliness - but forgot. Thanks for catching it.